### PR TITLE
ad9081: add zcu102 support

### PR DESCRIPTION
**Pull request by buha**
HDL design for zcu102 exists, this commit performs necessary
modifications to make no-OS/projects/ad9081 work on zcu102.

Signed-off-by: Darius Berghe <darius.berghe@analog.com>

Tested, works well on vcu118 + ad9081-fmca-ebz and zcu102 + ad9081-fmca-ebz.

### CODE REVIEW

File: drivers/adc/ad9081/ad9081.c [hunk 1]
```diff
@@ -707,18 +707,37 @@ static int32_t ad9081_setup(struct ad9081_phy *phy)
 	mdelay(10);
 
 	if (phy->jesd_rx_clk) {
-		mdelay(10);
-		ret = clk_enable(phy->jesd_rx_clk);
-		if (ret < 0) {
+		int timeout = 2000;
```
Comment (mchindri) [inline drivers/adc/ad9081/ad9081.c:710]: you can declare timeout at the beginning of the function and only initialize it here.
And also for consistency you should use int32_t instead of int.

File: drivers/adc/ad9081/ad9081.c [hunk 2]
```diff
@@ -707,18 +707,37 @@ static int32_t ad9081_setup(struct ad9081_phy *phy)
...
+		while(timeout) {
+			ret = clk_enable(phy->jesd_rx_clk);
+			if (ret) {
```
Comment (mchindri) [inline drivers/adc/ad9081/ad9081.c:713]: Can the changes in this file be in a separate commit?
"Add timeout for clk_enable " or something like that

File: projects/ad9081/src/app_parameters.h [hunk 3]
```diff
@@ -44,28 +44,48 @@
...
+#define PHY_CS			0
 
 #ifdef QUAD_MXFE
-#define ADF4371_CS	0
-#define HMC7043_CS	4
+#define ADF4371_CS		0
+#define HMC7043_CS		4
 
-#define PHY_RESET		41
+#define PHY_RESET		GPIO_OFFSET + 41
```
Comment (mchindri) [inline projects/ad9081/src/app_parameters.h:59]:
```suggestion
#define PHY_RESET		(GPIO_OFFSET + 41)
```

Comment (mchindri) [reply projects/ad9081/src/app_parameters.h:59]: same below

File: drivers/adc/ad9081/ad9081.c (hunk 1 above)
Comment (buha) [reply drivers/adc/ad9081/ad9081.c:710]: I prefer variables in the context where they are used but I think we have a rule about this so I'll move it at the top

File: drivers/adc/ad9081/ad9081.c (hunk 2 above)
Comment (buha) [reply drivers/adc/ad9081/ad9081.c:713]: Yeah good point

File: projects/ad9081/src/app_parameters.h (hunk 3 above)
Comment (buha) [reply projects/ad9081/src/app_parameters.h:59]: yeah, not really needed here but I'll do it anyway, it looks better

**Comment by buha** [comment]
V2:
- split the ad9081.c driver change into different commit, also moved timeout at the top of the function
- #define parantheses

**Comment by buha** [comment]
V3: 
- rebase on master (might get rid of ad77681 related astyle error)
